Tensile/Common.py

```diff
@@ -452,7 +455,11 @@
     {"UseSgprForGRO":             [ -1 ] },
     {"AssertSummationElementMultiple": [ 1 ] },
     {"AssertFree0ElementMultiple": [ 1 ] },
+<<<<<<< HEAD
```

Comment:
need to fix merge conflict

Reply:
I think this was cleaned up in a later checkin.  Given the scope of changes may be best to review final set.  The intermediate checkins are to help isolate failures if that becomes necessary